plugins/techdocs/src/reader/components/TechDocsPage.tsx

```diff
@@ -67,7 +71,7 @@ export const TechDocsPage = ({ children }: TechDocsPageProps) => {
   }, [setDocumentReady]);
 
   if (entityMetadataError) {
-    return <TechDocsNotFound errorMessage={entityMetadataError.message} />;
+    return <NotFoundPage errorMessage={entityMetadataError.message} />;
```

Comment:
@backstage/techdocs-core wondering if it makes sense to use the error pages that are configured as part of the app instead? Just to avoid too many duplicate configuration points if they're not needed https://github.com/backstage/backstage/blob/e90014359d6404a496264a49fed85b3451e9d49e/packages/core-app-api/src/app/types.ts#L69

Question for @jonathan-ash as well is whether using the globally configured error page would work for your use-case?

I'm feeling that the current message in the `TechDocsNotFound` page doesn't do much for the end user anyway, and is perhaps better surfaced through some other method?

Reply:
Missed the above discussion on this, so consider this a +1 @backstage/techdocs-core :grin: